llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -379,6 +381,30 @@ RISCVInstructionSelector::selectSHXADD_UWOp(MachineOperand &Root,
   return std::nullopt;
 }
 
+InstructionSelector::ComplexRendererFns
+RISCVInstructionSelector::selectVLOp(MachineOperand &Root) const {
```

Comment:
This isn't really selecting, it's an operand renderer. Call it render* not select*?